pest_meta/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "pest_meta"
+version = "1.0.0-beta.15"
+authors = ["Victor-Nicolae Savu <victor.nicolae.savu@gmail.com>"]
+repository = "https://github.com/pest-parser/pest"
+documentation = "https://docs.rs/pest"
+keywords = ["pest", "parser", "peg", "grammar"]
+categories = ["parsing"]
+license = "MPL-2.0"
+readme = "_README.md"
+
+[dependencies]
+quote = "^0.3"
+maplit = "^0.1.5"
```

Comment:
`maplit` [is just a bunch of macros](https://github.com/bluss/maplit/blob/master/src/lib.rs), so it doesn't really take any compilation time at all aside from downloading it (<200 lines of code including tests and documentation).

I thought you meant "quote", which is quite a bit of an annoyance since we only use one struct: `quote::Ident`. Of course, `quote` is already a dependency of `pest_derive`, so for that use case there's no added cost, but for projects who don't care much about producing a rust plugin it may be annoying.

Reply:
I meant both, actually. The only reason I'd give up on `maplit` is because it's easy to replace and one request less to worry about when compiling big projects with a lot of dependencies.

About `quote`, it may be a smart idea to give up on using `quote::Ident` in `pest_meta` since it doesn't make sense at all.

Reply:
I would add the circumvention of the `quote` crate as a second to do.